debian/nnstreamer-core.install

```diff
@@ -4,4 +4,5 @@
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_image_labeling.so
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_direct_video.so
 /usr/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
+/usr/lib/*/libnnstreamer.so
```

Comment:
Why don't you use `nnstreamer-core.links`

Reply:
Thanks. Changed!

Reply:
Uh.. I didn't see https://github.com/nnstreamer/nnstreamer/commit/1d1c9926bd3277a1724b3f52ea3bffb31422bc52